### PR TITLE
native: get height from webview to set editor container height in native

### PR DESCRIPTION
**Pull request by patosullivan**
and fix the flex issues with buttons (i.e., they stay at the bottom as the message input container grows)

fixes LAND-1813

### CODE REVIEW

File: packages/ui/src/components/MessageInput/index.native.tsx [hunk 1]
```diff
@@ -62,7 +67,7 @@ export function MessageInput({
   uploadedImage,
   canUpload,
 }: MessageInputProps) {
-  const [containerHeight, setContainerHeight] = useState(0);
+  const [containerHeight, setContainerHeight] = useState(52);
```
Comment (dnbrwstr) [inline packages/ui/src/components/MessageInput/index.native.tsx:70]: Would be nice to break this out as a constant and comment how it's derived.

File: packages/ui/src/components/MessageInput/index.native.tsx [hunk 2]
```diff
@@ -199,6 +199,29 @@ export function MessageInput({
...
+                const observer = new MutationObserver(updateContentHeight);
+                observer.observe(document.querySelector('#root'), { childList: true, subtree: true, attributes: true});
+
+                updateContentHeight(); // this sets initial height
+              }
+
+              // wait for the editor to load
+              setTimeout(() => {
+               setupMutationObserver();
+              }, 300);
```
Comment (dnbrwstr) [inline packages/ui/src/components/MessageInput/index.native.tsx:223]: Any way to wire this to a `ready` event or something like that? Timeouts just intrinsically flakey.

**Review by dnbrwstr — APPROVED** [review]
Lgtm! A couple small nits.

File: packages/ui/src/components/MessageInput/index.native.tsx (hunk 2 above)
Comment (patosullivan) [reply packages/ui/src/components/MessageInput/index.native.tsx:223]: Yeah we can send a ready event from the WebView side

**Comment by patosullivan** [comment]
Working on the ready event now

File: packages/ui/src/components/MessageInput/index.native.tsx [hunk 3]
```diff
@@ -148,12 +150,43 @@ export function MessageInput({
...
+        setContainerHeight(payload);
+        return;
+      }
+
       editor.bridgeExtensions?.forEach((e) => {
         e.onEditorMessage && e.onEditorMessage({ type, payload }, editor);
       });
     },
-    [editor, handleAddNewLine]
+    [editor, handleAddNewLine, webviewRef]
```
[the remark this answers is not in the document]
Comment (patosullivan) [reply packages/ui/src/components/MessageInput/index.native.tsx]: Yeah, I just added it to make the linter happy.

**Review by latter-bolden — APPROVED** [review]
Looking good!

**Pull request merged**